patches/chrome-browser-ui-views-tabs-tab_drag_controller.cc.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/chrome/browser/ui/views/tabs/tab_drag_controller.cc b/chrome/browser/ui/views/tabs/tab_drag_controller.cc
+index 705245efdc1736d54f066719415a84c8783072cf..a9747d316f49a3662339cdb28a3a91cf5f693333 100644
+--- a/chrome/browser/ui/views/tabs/tab_drag_controller.cc
++++ b/chrome/browser/ui/views/tabs/tab_drag_controller.cc
+@@ -491,6 +491,8 @@ void TabDragController::Init(TabDragContext* source_context,
+   last_move_screen_loc_ = start_point_in_screen_.x();
+   initial_tab_positions_ = source_context->GetTabXCoordinates();
+ 
++  BRAVE_TAB_DRAG_CONTROLLER_INIT_LOC_FOR_VERTICAL_TABS
```

Comment:
nit: We want to minimize patch file size also. Let's remove empty line adding below and from other patch files also.

Reply:
done.